packages/medusa/package.json

```diff
@@ -107,9 +108,6 @@
     "@medusajs/user": "^2.5.0",
     "@medusajs/workflow-engine-inmemory": "^2.5.0",
     "@medusajs/workflow-engine-redis": "^2.5.0",
-    "@swc/core": "1.5.7",
-    "@swc/helpers": "^0.5.11",
```

Comment:
Q: should we move the helper to dev delps?

Reply:
Ah I see it is a peer dependency, moving it 👍  It is already available but from another package